### DISCUSSION TITLE
About the Time Series

### DISCUSSION THREAD

**Discussion by itachiCheng** [PaddlePaddle/PaddleSpatial#72] [BaiduKDDCup2022] [answered]
We know that the time series of one Turbine is [day1 day2 day3...]. Are all the turbine similar with each other in the time line? For example, Is Turbine 1's [day1]  the same as Turbine 2's [day1]?

**Comment by moonquakes66** [comment]
I also want to ask.

**Comment by ramber1836** [comment, marked as answer]
All the turbines share exactly the same timeline.
